spyder/widgets/internalshell.py

```diff
@@ -408,10 +408,18 @@ def run_command(self, cmd, history=True, new_prompt=True):
         else:
             if history:
                 self.add_to_history(cmd)
-        self.interpreter.stdin_write.write(to_binary_string(cmd + '\n'))
         if not self.multithreaded:
-            self.interpreter.run_line()
-            self.refresh.emit()
+            if 'input' not in cmd:
+                self.interpreter.stdin_write.write(
+                                                to_binary_string(cmd + '\n'))
+                self.interpreter.run_line()
+                self.refresh.emit()
+            else:
+                self.write(_('In order to use commands like "raw_input" '
+                             'or "input" run Spyder with the multithread '
+                             'option (--multithread)'), error=True)

```

Comment:
Let's finish this message like this

    'option (--multithread) from a system terminal', error=True)